connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/DailyDigestSection.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.cloud.notifications.connector.email.model;
+
+import java.util.List;
+
+public class DailyDigestSection {
```

Comment:
Isn't this a duplicate of `DailyDigestSection` from `common-template`? If so, can we avoid duplicating it?